### ISSUE TITLE
Page goes blank while redirecting to destination link

### ISSUE THREAD

**Issue by monizb** [monizb/FireShort#74]
**Describe the bug**
When the short URL is accessed and the Fireshort Loader is presented, it disappears for a few seconds before redirecting to thr destination link

**To Reproduce**
1. Go to the shortened URL
2. Wait for a second
3. Issue Occurs

**Expected behavior**
Show the Loader until the page is redirected

**Screenshots**

This issue has occurred after #46 was merged, @harshvats2000 please take a look at this if you are interested :)

**Comment by TaarnStar** [comment]
Hi @monizb I would like to try fix this, can you assign me?

**Comment by monizb** [comment]
@TaarnStar glad to know you want to contribute! Please go ahead

**Comment by TaarnStar** [comment]
@monizb I successfuly started the project, and added one short link. However it seems I cannot reproduce the issue. When I click on the link of the short link (on Open option), it opens the short link immediately (in the new tab), I don't see the Fireshort Loader.

I don't know if I am doing something wrong so would appreciate your help in figuring out how to reproduce the issue so I can resolve it

**Comment by TaarnStar** [comment]
@monizb I was able to reproduce the issue by copying the link and not clicking on the option open, I will get right on fixing the bug

**Comment by monizb** [comment]
@TaarnStar sure, no issues, Open function is meant to take you to the original link directly :)

**Comment by TaarnStar** [comment]
@monizb I think I found the issue and I fixed it, the trouble is, when I try to go to a link that requires a password I get another error that data is undefined (when to the realPassword gets assigned data.password) so I cannot test it properly (the console error is there regardless if I change anything or just start the default app), also I looked into the pull request [#46](https://github.com/monizb/FireShort/pull/46) where the screenshots of the app aren't the same as I see the app (on the screenshots there is an icon which shows if the link is secured or not but on ma local app there isn't any icon like that).

I thought maybe it was the wrong branch, but I saw that the PR was merged into the master branch which I forked. Do you know if I have the wrong branch perhaps?

**Comment by monizb** [comment]
@TaarnStar the #46 features were merged into master and I had fixed up the UI to look better,maybe that's the reason you find it different, please do make sure you are upto date with the master branch because there are a lot of PRs being merged right now.Also please raise a issue with appropriate screenshots and error and I will help you out with it

**Comment by TaarnStar** [comment]
@monizb I was able to fix and test the issue. In short the problem I had was that my short link had spaces in it's name so that's why there was error in console. Do you think that is something that should be fixed (maybe by not allowing to name a short link with spaces)? If yes I will open the issue (and if it is ok will start working on the fix too).

I made the [PR](https://github.com/monizb/FireShort/pull/78) for this issue, if there is the need to change anything regarding the fix don't hesitate to tell me :)
